### ISSUE TITLE
Hyperlinks don't function in nested tabs.

### ISSUE THREAD

**Issue by josephbr** [wet-boew/wet-boew#6539]
I currently have a page containing nested tabs. Each parent tab contains the Year and all the child ones have the months (for that year). However, for some reason, hyperlinks do not function when placed within a child tab. Clicking them just advances to the next child tab rather than directing the browser to the URL. Bear in mind that these are just ordinary hyperlinks/anchor tags; nothing special about them.

Is WET not set up to support nested tabs?

**Comment by pjackson28** [comment]
Nested tabs have been tested successfully in the past (although fixes were needed a few versions ago). What version of WET are you using? Please post your markup as well so others can attempt to reproduce.

**Comment by josephbr** [comment]
I tried it with v4.0.10, v.4.0.11, and the latest dev build as of when I first posted this ticket and all of them had the same problem.

I managed to re-produce the bug using the following markup. Just  go to "Outer Tab 2" and the nested "Inner Tab 1" and then click the "CLICK ME!" link. Instead of going to the pages it links to (for the sake of testing, I used http://www.google.ca), it instead advances to "Inner tab 2". This happens with ALL links I link inside a nested tab.

```
<div class="wb-tabs">
  <div class="tabpanels">
    <details id="details-panel1">
      <summary>Outer Tab 1</summary>
      <p>Test One</p>
    </details>
    <details id="details-panel2" open="open">
      <summary>Outer Tab 2</summary>
      <div class="wb-tabs">
        <div class="tabpanels">
          <details id="details-panel3">
            <summary>Inner Tab 1</summary>
            <p>Test Link Tab</p>
            <a href="http://www.google.ca">CLICK ME!</a>
          </details>
          <details id="details-panel5" open="open">
            <summary>Inner Tab 2</summary>
            <p>Test Four</p>
          </details>
        </div>
      </div>
    </details>
  </div>
</div>
```

**Comment by pjackson28** [comment]
Will be fixed by #6559. Turns out that it was a combo tabs/toggle bug.
